metrics/machine_learning/README.md

```diff
@@ -0,0 +1,16 @@
+# Kata Containers Tensorflow Metrics
+
+Kata Containers provides a series of performance tests using the
+TensorFlow reference benchmarks (tf_cnn_benchmarks).
+The tf_cnn_benchmarks containers TensorFlow implementations of several
```

Comment:
Nit: I think you might need to put the `tf_* terms in back ticks to keep the spell checker happy.